src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -24,36 +26,46 @@
+        .withPhone("94351253").withId("cfa7fa72-e310-4928-a53b-2a5658242f7b")
             .withTags("friends").build();
     public static final Person BENSON = new PersonBuilder().withName("Benson Meier")
-            .withAddress("311, Clementi Ave 2, #02-25")
-            .withEmail("johnd@example.com").withPhone("98765432")
-            .withTags("owesMoney", "friends").build();
+        .withAddress("311, Clementi Ave 2, #02-25").withId("5ad1dd04-e8d8-411e-9916-2f89bce26a85")
+        .withEmail("johnd@example.com").withPhone("98765432")
+        .withTags("owesMoney", "friends").build();
     public static final Person CARL = new PersonBuilder().withName("Carl Kurz").withPhone("95352563")
-            .withEmail("heinz@example.com").withAddress("wall street").build();
+        .withEmail("heinz@example.com").withAddress("wall street")
+        .withId("5ad1dd04-e8d8-3333-9916-2f89bce26b85").build();
     public static final Person DANIEL = new PersonBuilder().withName("Daniel Meier").withPhone("87652533")
-            .withEmail("cornelia@example.com").withAddress("10th street").withTags("friends").build();
+        .withEmail("cornelia@example.com")
+        .withId("5ad1dd04-e8d8-411e-9916-2f89bce26b83")
+        .withAddress("10th street").withTags("friends").build();
     public static final Person ELLE = new PersonBuilder().withName("Elle Meyer").withPhone("9482224")
-            .withEmail("werner@example.com").withAddress("michegan ave").build();
+        .withEmail("werner@example.com")
+        .withId("5ad1dd04-2222-311e-9916-2f89bce26b85")
+        .withAddress("michegan ave").build();
     public static final Person FIONA = new PersonBuilder().withName("Fiona Kunz").withPhone("9482427")
-            .withEmail("lydia@example.com").withAddress("little tokyo").build();
-    public static final Person GEORGE = new PersonBuilder().withName("George Best").withPhone("9482442")
-            .withEmail("anna@example.com").withAddress("4th street").build();
+        .withEmail("lydia@example.com")
+        .withId("5ad1dd04-e8d8-411e-9916-2f89bce26baa")
+        .withAddress("little tokyo").build();
+    public static final Person GEORGE = new PersonBuilder().withName("George Best")
+        .withId("5ad1dd04-e8d8-411e-9916-2f89bce26bcc")
+        .withPhone("9482442")
+        .withEmail("anna@example.com").withAddress("4th street").build();
 
     // Manually added
     public static final Person HOON = new PersonBuilder().withName("Hoon Meier").withPhone("8482424")
-            .withEmail("stefan@example.com").withAddress("little india").build();
+        .withEmail("stefan@example.com").withAddress("little india").build();
     public static final Person IDA = new PersonBuilder().withName("Ida Mueller").withPhone("8482131")
-            .withEmail("hans@example.com").withAddress("chicago ave").build();
+        .withEmail("hans@example.com").withAddress("chicago ave").build();
```

Comment:
I noticed that there is no withId(...) used for these 2 persons. Are they intended to be used with defaultId?